package.json

```
@@ -27,18 +27,18 @@
 	"homepage": "https://github.com/ckeditor/ckeditor4-integrations-common#readme",
 	"browserslist": "> 0.25%, not dead",
 	"devDependencies": {
-		"@babel/core": "^7.11.6",
-		"@babel/preset-env": "^7.14.1",
+		"@babel/core": "^7.17.10",
+		"@babel/preset-env": "^7.17.10",
 		"@rollup/plugin-babel": "^5.3.0",
-		"babel-loader": "^8.1.0",
+		"babel-loader": "^8.2.5",
 		"chai": "^4.2.0",
-		"eslint": "^8.10.0",
+		"eslint": "^8.14.0",
```

```suggestion
		"eslint": "^8.16.0",
```
There is a newer version of `eslint`.